remediation/machine-deletion/machine-deletion.md

```diff
@@ -0,0 +1,13 @@
+---
+layout: default
+title: Machine Deletion
+nav_order: 3
+has_children: true
+parent: Remediation
+---
+
+# Machine-API Driven Remediation
+Generally Available
+{: .label .label-green }
+
+This operator conforms to the External Remediation of [NodeHealthCheck](https://github.com/medik8s/node-healthcheck-operator#readme) and is designed to work with [Node Health Check]((https://github.com/medik8s/node-healthcheck-operator#readme)) to reprovision unhealthy nodes using the [Machine API](https://github.com/openshift/machine-api-operator#readme). It functions by following the annotation on the Node to the associated Machine object, confirms that it has an owning controller (e.g. MachineSetController), and deletes it.  Once the Machine CR has been deleted, the owning controller creates a replacement. 
```

Comment:
- can you add some line breaks please for better readability in the IDE :)
- the 2nd link is broken (double braces around URL)
- NHC is mentioned twice directly after another, that doesn't sound nice IMHO ("conforms to... and is designed to")
- "It functions by following the annotation on the Node to the associated Machine object" ... what annotation is followed for doing what? 🤔 Edit: ok, after reading it a few times more I get it... but think that's an unimportant implementation detail at least at this point. Maybe just something like "it looks for the associated Machine of an unhealthy Node..."?

Reply:
Thanks, much appreciated !